bin/check-package-metadata.js

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script that will verify that all packages in our monorepo
+ * are registered in all required places and have the expected metadata in their
+ * package.json.
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+
+const Project = require('@lerna/project');
+
+const MIT_LICENSE_TEXT = `
+MIT license
```

Comment:
On the one hand, I’m not sure about this type of check, on the other hand, the license file will be the same everywhere.

Reply:
For our monorepo, we run `lb4 copyright` command once a while to make sure the license headers are up to date.

Reply:
Maybe I can take root folder LICENSE file as source of truth, but in this case loopback needs some way how to support this file in actual state? Or I can simplify check for license file

Reply:
Or another solution: take package `spdx-license-list` which is using in `packages/cli`. Anyway, guys, I need your decision what is the sufficient way from your perspective? @raymondfeng @bajtos

Reply:
Checking the existence of `LICENSE` file is probably good enough as `lb4 copyright` can deal with the consistency.